static/src/javascripts/projects/journalism/views/campaignForm.html

```diff
@@ -58,7 +58,6 @@ <h4 class="campaign--snippet__headline"> <%= data.title %></h4>
                        value="<%=option.value%>"
                        id="<%=option.label%>"
                        name="field_<%=field.id%>"
-                    <% if (field.required == "1") { %> required <% } %>
```

Comment:
Are we saying that required fields can't apply to checkbox and radio buttons?

Reply:
yes, esp. not when there are more than one (which is always the case). browsers aren't smart enough to know that _at least_ one in the group should be ticked, not all.